main/ejml-dsparse/test/org/ejml/sparse/csc/mult/TestMatrixVectorMultWithSemiRing_DSCC.java

```diff
@@ -0,0 +1,124 @@
+    public void mult_v_A(String desc, DSemiRing semiRing, double[] expected) {
+        // graphblas == following outgoing edges of source nodes
+        double v[] = new double[7];
+        Arrays.fill(v, semiRing.add.id);
+        v[3] = 0.5;
+        v[5] = 0.6;
+
+        System.out.println("input vector = " + Arrays.toString(v));
+
+        //input.print();
+
+        double found[] = new double[7];
+
+        MatrixVectorMultWithSemiRing_DSCC.mult(v, inputMatrix, found, semiRing);
+
+        System.out.println("found = " + Arrays.toString(found));
+        System.out.println("expected = " + Arrays.toString(expected));
+        assertTrue(Arrays.equals(found, expected));
+    }
+
+    @ParameterizedTest(name = "{0}")
+    @MethodSource("matrixVectorMultSources")
+    public void mult_A_v(String desc, DSemiRing semiRing, double[] expected) {
+        // graphblas == following incoming edges of source nodes
+        double v[] = new double[7];
+        Arrays.fill(v, semiRing.add.id);
+        v[3] = 0.5;
+        v[4] = 0.6;
+
+        System.out.println("input vector = " + Arrays.toString(v));
+
+        //input.print();
+
+        double found[] = new double[7];
+
+        MatrixVectorMultWithSemiRing_DSCC.mult(inputMatrix, v, found, semiRing);
+
+        System.out.println("found = " + Arrays.toString(found));
+        System.out.println("expected = " + Arrays.toString(expected));
+        assertTrue(Arrays.equals(found, expected));
+    }
+
+    private static Stream<Arguments> vectorMatrixMultSources() {
+        return Stream.of(
+                Arguments.of("Plus, Times", DSemiRings.PLUS_TIMES, new double[]{0.1, 0, 0.5, 0, 0, 0, 0}),
+                Arguments.of("OR, AND", DSemiRings.OR_AND, new double[]{1, 0, 1, 0, 0, 0, 0}),
+                Arguments.of("MIN, PLUS", DSemiRings.MIN_PLUS,
+                        new double[]{0.7, Double.MAX_VALUE, 0.9, Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE}),
+// This only works on sparse input vectors (here resulting in 1.0 instead of Double.MIN_Value as max(1.0, Double.MIN_VALUE) = 1.0)
+//                Arguments.of("MAX, PLUS", PreDefinedDoubleSemiRings.MAX_PLUS,
+//                        new double[]{0.7, Double.MIN_VALUE, 1.1, Double.MIN_VALUE, Double.MIN_VALUE, Double.MIN_VALUE, Double.MIN_VALUE}),
+                Arguments.of("MIN, TIMES", DSemiRings.MIN_TIMES,
+                        new double[]{0.1, Double.MAX_VALUE, 0.2, Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE}),
+                Arguments.of("MAX, MIN", DSemiRings.MAX_MIN,
+                        new double[]{0.2, Double.MIN_VALUE, 0.5, Double.MIN_VALUE, Double.MIN_VALUE, Double.MIN_VALUE, Double.MIN_VALUE})
+        );
+    }
+
+    private static Stream<Arguments> matrixVectorMultSources() {
+        return Stream.of(
+                Arguments.of("Plus, Times", DSemiRings.PLUS_TIMES, new double[]{0.5, 0.6, 0, 0, 0, 0, 1.1}),
```

Comment:
Why not `"PLUS, TIMES"` here and in other usages of `Arguments`?

Reply:
Good point. Will make it more consistent with the other arguments :+1: